ui/app/templates/components/dashboard/replication-card.hbs

```diff
@@ -0,0 +1,177 @@
+<div class="is-flex-between">
+  <h3 class="title is-4 has-bottom-margin-xxs" data-test-client-count-title>
+    Replication
+  </h3>
+
+  <div>
+    <Hds::Link::Inline @route="vault.cluster.replication.index" class="is-no-underline">Details</Hds::Link::Inline>
```

Comment:
Looking at the [docs](https://helios.hashicorp.design/components/link/standalone) I think these should all be standalone links 🤔

Reply:
@hellobontempo Yeah, my initial thought was to use standalone links but I noticed [standalone links](https://helios.hashicorp.design/components/link/standalone?tab=code) require an icon. I might have misread it, so please let me know if I did! :)

Reply:
I think we can use the native `LinkTo` component that comes with ember to avoid the extra divs